packages/block/.eslintrc.cjs

```diff
@@ -1 +1,15 @@
-module.exports = require('../../config/eslint.cjs')
+module.exports = {
+    extends: '../../config/eslint.cjs',
+    parserOptions: {
+      project: ['./tsconfig.lint.json'],
+    },
+    overrides: [
+      {
+        files: ['examples/**/*'],
+        rules: {
+          'no-console': 'off',
+          '@typescript-eslint/no-unused-vars': 'off',
+        },
```

Comment:
Why do we need to overwrite the examples folder for every package? Isn't there a way to specify that generally?

Reply:
Let me experiment and see.